### PR TITLE
sweave2rmd: Convert Accept-organism-for-GenomeInfoDb.Rnw to .Rmd

### PR DESCRIPTION
**Pull request by hsadia538**
Part of Sweave2rmd project.

- Create a vignette in R markdown which replaces the sweave vignette.
- Fix Spellings

Attached are the HTML vignette and Sweave for comparison here.
[Rnw File](http://bioconductor.org/packages/devel/bioc/vignettes/GenomeInfoDb/inst/doc/Accept-organism-for-GenomeInfoDb.pdf)
[Rmd File](https://github.com/Bioconductor/GenomeInfoDb/files/9889532/Submitting.your.organism.to.GenomeInfoDb.html.zip)

### CODE REVIEW

File: vignettes/Accept-organism-for-GenomeInfoDb.Rmd [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+- name: "Sonali Arora"
+- name: "H. Khan"
+  affiliation: "Vignette translation from Sweave to Rmarkdown / HTML"
+date: "`r format(Sys.time(), '%d %B, %Y')`"
+package: GenomeInfoDb
+vignette: >
+  %\VignetteIndexEntry{Submitting your organism to GenomeInfoDb}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+output:
```
Comment (mcarlsn) [inline vignettes/Accept-organism-for-GenomeInfoDb.Rmd:13]: Small edit - Please change to include TOC using this code:

output:
  BiocStyle::html_document:
    number_sections: yes
    toc: yes
    toc_depth: 4

File: vignettes/Accept-organism-for-GenomeInfoDb.Rmd [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+```{r message=FALSE}
+library(GenomeInfoDb)
+names(genomeStyles())
+```
+
+# File format for new organism
+
+If your favorite organism does not exist in the above list, one can submit a tab-delimited file in the following format and request for your organism to be added to [*GenomeInfoDb*](http://bioconductor.org/packages/release/bioc/html/GenomeInfoDb.html)
+
+1.  The file name should be in the following format:
```
Comment (mcarlsn) [inline vignettes/Accept-organism-for-GenomeInfoDb.Rmd:36]: After "in the following format", please add the following text to match the original vignette:  ‘genus\_{}species.txt’

**Review by mcarlsn — CHANGES_REQUESTED** [review]
Great work. In addition to these 2 comments, it looks like the DESCRIPTION file does not have `BiocStyle` and `knitr` under `Suggests`. Thank you for your work and time converting this vignette!

**Review by hsadia538 — COMMENTED** [review]
@mcarlsn I can see BiocStyle and knitr under suggests

File: vignettes/Accept-organism-for-GenomeInfoDb.Rmd (hunk 2 above)
Comment (hsadia538) [reply vignettes/Accept-organism-for-GenomeInfoDb.Rmd]: I have updated the changes

**Comment by jwokaty** [comment]
@hpages Please review and accept if it is ok or let me know if you want any changes. Thanks.

**Pull request merged**